### ISSUE TITLE
add low-bandwidth mode

### ISSUE THREAD

**Issue by KiaraGrouwstra** [LemmyNet/jerboa#1594]
### Pre-Flight checklist

- [X] Did you check to see if this issue already exists?
- [X] This is a single feature request. (Do not put multiple feature requests in one issue)
- [X] This is not a question or discussion. (Use https://lemmy.ml/c/jerboa for that)

### Describe The Feature Request Below

jerboa's list view currently eagerly loads links' preview images.
on metered connections, it may be desirable to disable this so as to conserve bandwidth.
as such, it would be nice to see such a low-bandwidth mode added to jerboa.

c.f. #584 which mentioned this consideration as well

**Comment by MV-GH** [comment]
related #716

**Comment by dessalines** [comment]
A low-bandwidth mode could also decrease the thumbnail size when requesting from pictrs, that'd be easy enough.

**Comment by KiaraGrouwstra** [comment]
> A low-bandwidth mode could also decrease the thumbnail size when requesting from pictrs, that'd be easy enough.

for what it's worth, this particular suggestion was not implemented yet in #2111, and could make for a potential follow-up.

**Comment by MV-GH** [comment]
I don't think it would be worth it, its only the image viewer that would still show images and showing there then a low res image wouldn't be particularly useful

**Comment by dessalines** [comment]
@MV-GH lmk if I can do a release, or if you want me to include anything else.

**Comment by MV-GH** [comment]
I don't have anything planned, so feel free to do it whenever you prefer

**Comment by MV-GH** [comment]
I am starting to think its was a bad idea to include "metered" in auto setting.

see this definition

> A metered network is an internet connection with a specific data cap (daily or monthly), where users pay for the amount of data used or face throttling/extra fees upon exceeding a limit, similar to mobile phone data plans. Setting a connection to "metered"

So this would include every (mobile) network with a cap? Isn't that most networks?

I think it would be better if it we would only look at the "Data Saving" on part of the phone

Cuz I am seeing lots of reports of people not being aware this is active.
